util/dvsim/CfgJson.py

```diff
@@ -167,7 +169,13 @@ def set_target_attribute(path, target, key, dict_val):
         target[key] = dict_val
         return
 
+    # both fields are not default. Always uses leaf most field and raise warnings rather than RuntimeError
+    target[key] = old_val
+    log.warning('{!r}: Value for key {!r} is {!r}, but '
+                'we already had a conflicting value of {!r}.'
+                .format(path, key, dict_val, old_val))
+
     # Neither value looks like a default. Raise an error.
-    raise RuntimeError('{!r}: Value for key {!r} is {!r}, but '
-                       'we already had a conflicting value of {!r}.'
-                       .format(path, key, dict_val, old_val))
+    #raise RuntimeError('{!r}: Value for key {!r} is {!r}, but '
+    #                   'we already had a conflicting value of {!r}.'
+    #                   .format(path, key, dict_val, old_val))
```

Comment:
I think you can just delete these lines, the git history will be there if we need to revert it ever.

Reply:
Done :)